contracts/interfaces/modules/royalty/IRoyaltyModule.sol

```diff
@@ -30,9 +30,22 @@ interface IRoyaltyModule is IModule {
     /// @param amount The amount paid
     event LicenseMintingFeePaid(address receiverIpId, address payerAddress, address token, uint256 amount);
 
+    /// @notice Sets the licensing module
+    /// @dev Enforced to be only callable by the protocol admin
+    /// @param licensing The address of the license module
+    function setLicensingModule(address licensing) external;
+
+    /// @notice Sets the dispute module
+    /// @dev Enforced to be only callable by the protocol admin
+    /// @param dispute The address of the dispute module
+    function setDisputeModule(address dispute) external;
+
     /// @notice Returns the licensing module address
     function licensingModule() external view returns (address);
 
+    /// @notice Returns the dispute module address
+    function disputeModule() external view returns (address);
+
```

Comment:
These two can also be removed, as we don't expect the functions to be called by another contract.